tests/Aspire.Hosting.Tests/Azure/AzureBicepResourceTests.cs

```diff
@@ -494,7 +553,7 @@ public async Task PublishAsRedisPublishesRedisAsAzureRedisConstruct()
                 value: '${redisCache_p9fE6TK3F.properties.hostName},ssl=true,password=${redisCache_p9fE6TK3F.listKeys(redisCache_p9fE6TK3F.apiVersion).primaryKey}'
               }
             }
-            
+
```

Comment:
What are these whitespace changes about? This concerns me it means your generation was different to what was there previously. Did CDK change some whitespace?

Reply:
These are in the tests and not part of the payload. I think it was just my IDE settings.